scripts/export-rust-source.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -e # Errors are fatal
+set -x # Show commands
+
+OBJDIR="$1"
+
+mkdir -p "$OBJDIR/__RUST__"
+
+cp -r "$(rustc --print sysroot)/lib/rustlib/src/rust/src" "$OBJDIR/__RUST__"
```

Comment:
Shouldn't the destination have a `__GENERATED__` in it?

Reply:
Maybe we want to create a fourth category beyond normal/test/generated of "stdlib"?

Reply:
> Sure. Though that can be compatible to still have the same setup right? Do you know where we generate the data?

https://github.com/mozsearch/mozsearch/blob/master/router/router.py#L121 near the bottom checks for "__GENERATED__" to return generated.